hack/common-vars.sh

```diff
@@ -13,7 +13,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 # shellcheck disable=SC2034
 KUBECTL="${REPO_ROOT}/hack/tools/bin/kubectl"
```

Comment:
What about these variables that assume `REPO_ROOT` is set?

Reply:
A check to verify that assumption seems worthwhile. The pattern right now seems to be setting `$REPO_ROOT` in each script and using that to source `common-vars.sh`, so it should always be set here.

Reply:
nevermind, see the comment in the ticket description.

Reply:
Added a check since it's entirely possible future changes could source common-vars.sh without having already defined `$REPO_ROOT`.